shared/src/main/scala/drt/shared/Api.scala

```diff
@@ -579,19 +592,26 @@ object FlightsApi {
 
   case class FlightsWithSplits(flightsToUpdate: List[ApiFlightWithSplits], arrivalsToRemove: List[Arrival]) extends PortStateMinutes {
     def applyTo(portState: PortStateMutable, now: MillisSinceEpoch): PortStateDiff = {
+      val minutesFromRemovals: List[MillisSinceEpoch] = arrivalsToRemove.flatMap(r =>
+        portState.flights.getByKey(r.unique).map(_.apiFlight.pcpRange().toList).getOrElse(List())
+      )
+      val minutesFromUpdates = flightsToUpdate.flatMap(_.apiFlight.pcpRange())
```

Comment:
add a test to ensure that flight removals result in the correct minutes being flagged for recrunch.

Reply:
Thankfully this is covered by an existing test in `RunnableDeskRecsSpec`:
```
  "Given a flight with splits which is subsequently removed " +
    "When I ask for the output loads " +
    "Then I should see zero loads for the minutes affected by the flight in question"
```